modules/ccs-gcp-customer-procedure.adoc

```diff
@@ -16,7 +16,7 @@ To use {product-title} in your GCP project, the following GCP organizational pol
 
 * `constraints/iam.allowedPolicyMemberDomains`
 * `constraints/compute.restrictLoadBalancerCreationForTypes`
-* `constraints/compute.requireShieldedVm`
+* `constraints/compute.requireShieldedVm`  (This policy constraint is unsupported only if the cluster is installed without selecting "Enable Secure Boot support for Shielded VMs" during the initial cluster creation).
```

Comment:
And I recognize that this would make the wording inconsistent with the next bullet point, but I don't get why that one doesn't say "This policy constraint is supported only after installation." in the first place. Though that's out of scope for my review

Reply:
@svmrh,
      Please see comment above. Any thoughts on this suggestion? Sebastian is referring to the constraint as identified in the image below. Thanks! 
![image](https://github.com/openshift/openshift-docs/assets/122639474/99f624f9-39cc-4ba7-8f5c-5b64d5bdb692)